### ISSUE TITLE
Ubuntu 20.04.2 LTS - script stop with error

### ISSUE THREAD

**Issue by yboujraf** [Yenthe666/InstallScript#311]
Dear @Yenthe666 

**Information**

`$ lsb_release -a`

```
No LSB modules are available.     
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.2 LTS
Release:        20.04
Codename:       focal
```

**Error**

```
$ sudo apt-get install git python3 python3-pip build-essential wget python3-dev python3-venv python3-wheel libxslt-dev libzip-dev libldap2-dev libsasl2-dev python3-setuptools node-less libpng12-0 libjpeg-dev gdebi -yess libpng12-0 libjpeg-dev gdebi -y
[sudo] password for odoo14:
Reading package lists... Done
Building dependency tree
Reading state information... Done
Note, selecting 'libxslt1-dev' instead of 'libxslt-dev'
Package libpng12-0 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libpng12-0' has no installation candidate
```

Finally, Python is not installed but the script continue to be installed with errors.

Best Regards

**Comment by Yenthe666** [comment]
It is added here before trying to install it:
https://github.com/Yenthe666/InstallScript/blob/00adb8fb5490bbeb4164eefe29e3de90ee51af05/odoo_install.sh#L61
What do you get when you run that command?

**Comment by yboujraf** [comment]
The command is not executed.

Best Regards,

Youssef BOUJRAF

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by Yenthe666** [comment]
Euhh?? What do you get if you manually run it in the terminal?

**Comment by yboujraf** [comment]
Dear @Yenthe666 
I am using the version focal and not xenial?

BR

**Comment by chris001** [comment]
@yboujraf 
Can you run this command from the terminal and paste back the result:
`sudo add-apt-repository "deb http://mirrors.kernel.org/ubuntu/ focal main"`

**Comment by yboujraf** [comment]
Dear @chris001 

I added the line :

![image](https://user-images.githubusercontent.com/6151258/118237710-dfa31380-b497-11eb-928c-b03a12bda00d.png)

Best Regards,

**Comment by chris001** [comment]
@yboujraf Looks like the command works!
So now, if you edit the script (line 61), replace the word `xenial` with `focal`, save it, and re-run the script, does it work without errors? Paste back the output of re-running your script here!

**Comment by holdenrehg** [comment]
I just setup a brand new Ubuntu 20.04 droplet on Digital Ocean, copied the script as-is, and ran it without any errors. Not sure if these specific errors have been resolved with recent updates to 20.04 or not, but throwing this out there for reference.

**Comment by Yenthe666** [comment]
I'll consider it solved then. Thanks for letting me know Rehg :)
